src/alfasim_score/converter/alfacase/score_input_reader.py

```diff
@@ -177,6 +179,25 @@ def read_casings(self) -> List[Dict[str, Any]]:
                         "final_md": Scalar(item["final_md"], LENGTH_UNIT, "length"),
```

Comment:
"""Read geometric data for the casing and annulus data associated to the casing from SCORE input file."""